src/counter.cpp

```diff
@@ -369,6 +369,11 @@ void Counter::set_up_probs_threshold_measurements(
 
     verb_print(1, "[appmc] threshold set to " << threshold << " sparse: " << (int)using_sparse);
 
+    // Error probability of ApproxMC6
+    // See Lemma 4 in CAV23 paper "Rounding Meets Approximate Model Counting"
+    // https://link.springer.com/chapter/10.1007/978-3-031-37703-7_7
+
+    // Upper bound on the probability of underestimation event (L)
     double p_L = 0;
```

Comment:
``
```suggestion
    double p_L;
```

No need to set to 0, it's set below in the `else`.